app/build.gradle

```diff
@@ -24,9 +24,10 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "com.android.support:support-annotations:$rootProject.supportLibraryVersion"
-    implementation "com.android.support:appcompat-v7:$rootProject.supportLibraryVersion"
```

Comment:
As stated in the previous review which you said double checked, don't remove appcompact library here.